frontend/public/features.ts

```diff
@@ -155,11 +155,24 @@ const detectMonitoringURLs = dispatch => coFetchJSON(monitoringConfigMapPath)
     },
   );
 
+const detectUsername = dispatch => coFetchJSON('api/kubernetes/apis/user.openshift.io/v1/users/~')
+  .then(
+    (user) => {
+      dispatch(UIActions.setUsername(_.get(user, 'fullName') || user.metadata.name));
```

Comment:
We can't reliably check for `kube:admin` this way because we don't know if `fullName` will ever be set for that user.

I would store the entire user object in redux, then we can check specifically for `metadata.name` to show the banner. (You'd probably want to change `username` to `user` in all the names.)